src/value_generators/src/DocumentGenerator.cpp

```diff
@@ -488,7 +510,11 @@ UniqueExpression RandomStringExpression::parse(YAML::Node node) {
         }
     }
 
-    return std::make_unique<RandomStringExpression>(std::move(length), std::move(alphabet));
+    UniqueTypedExpression<IntegerValueType> lengthT =
+        std::make_unique<TypedExpression<IntegerValueType>>(std::move(length));
+
+    // TODO: does this want to be `optional`?
```

Comment:
What is "this"?

Reply:
lol can't remember my thought process - just removed that todo